hc/front/tests/test_update_timeout.py

```diff
@@ -57,3 +57,22 @@ def test_it_checks_ownership(self):
         self.client.login(username="charlie@example.org", password="password")
         r = self.client.post(url, data=payload)
         assert r.status_code == 403
+        
+    def test_update_timeout_and_grace_works(self):
```

Comment:
Good implementation of the test, although add a docstring

Reply:
Okay,Thanks